autoload/coverage/vim.vim

```diff
@@ -0,0 +1,28 @@
+" Copyright 2017 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+"{{{ covimerage provider (based on coverage.py)
```

Comment:
We should get a list of the built-in providers into the help now that there are 2. If you prefer, we could do that as a separate change after you submit this, but if so could you in the meantime add a link here to information about covimerage?

Reply:
Added (amended).

Reply:
Will create a follow-up PR later with docs.
I've found that docs are in 3 places: README.md, doc/coverage.txt, and instant/flags.vim - and its not consistent:
1. doc/neomake.txt mentions coverage#AddProvider (which does not exist), so I guess it is wrong?
2. what is the doc in instant/flags.vim for? Is this for Maktaba?

Reply:
I don't see a doc/neomake.txt in the repo. Maybe that's something that got generated locally on your system somehow?

The doc comments in instant/flags.vim are used via the tool http://github.com/google/vimdoc to generate parts of doc/coverage.txt.  They could be in other files, but by convention we put the misc doc sections in instant/flags.vim if there is one since that's sort of the entry point and always has some vimdoc comments.